src/sql/parts/objectExplorer/common/objectExplorerService.ts

```diff
@@ -439,14 +439,19 @@ export class ObjectExplorerService implements IObjectExplorerService {
 						finalResult = result;
 						allNodes = allNodes.concat(result.nodes);
```

Comment:
Just realized we're missing handling of a null/undefined nodes result. Won't it add an undefined node here? Can we just put a quick guard in against that? Thanks.